### PR TITLE
[DOC] - Update Tutorials for 2.2

### PR DESCRIPTION
**Pull request by TomDonoghue**
This PR add & extends tutorials related to additions for the 2.2 release.

### CODE REVIEW

File: tutorials/sim/plot_SimulateAperiodic.py [hunk 1]
```diff
@@ -166,3 +167,84 @@
...
+# :func:`~.sim_frac_brownian_motion` function.
+#
+
+###################################################################################################
+
+# Simulate fractional brownian motion signal
+bm_ap = sim_frac_brownian_motion(n_seconds, fs, exponent=-2)
+
+# Plot the simulated data, in the time domain
+plot_time_series(times, gn_ap, title='Simulated Fractional Brownian Motion')
```
Comment (ryanhammonds) [inline tutorials/sim/plot_SimulateAperiodic.py:250]: gn_ap -> bm_ap

File: tutorials/sim/plot_SimulateCombined.py [hunk 2]
```diff
@@ -152,3 +153,41 @@
 
 # Plot the simulated data, in the time domain
 plot_time_series(times, sig)
+
+###################################################################################################
+# Simulate Peak Oscillation
+# -------------------------
+#
+# Next, we will simulate time series with a peak in the power spectrum, that we can
```
Comment (ryanhammonds) [inline tutorials/sim/plot_SimulateCombined.py:161]: simulate {a} time series

File: tutorials/sim/plot_SimulateCombined.py [hunk 3]
```diff
@@ -152,3 +153,41 @@
...
+
+###################################################################################################
+
+# Precompute an aperiodic time series
+ap_sig = sim_powerlaw(n_seconds, fs, exponent=-1)
+
+# Define settings that define the peak to add
+freq = 10
+bw = 3
+height = 1
```
Comment (ryanhammonds) [inline tutorials/sim/plot_SimulateCombined.py:179]: Maybe increase height so the oscillatory peak is more distinct? Or increase n_seconds so spectra is a bit more stable. I'll play around with this.

File: tutorials/sim/plot_SimulateModulated.py [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+###################################################################################################
+
+# Plot the amplitude modulated signal
+plot_time_series(times, msig)
+
+###################################################################################################
+# Call signatures
+# ^^^^^^^^^^^^^^^
+#
+# In the above, we have explicitly simulating both the base signal and the modulating signal.
```
Comment (ryanhammonds) [inline tutorials/sim/plot_SimulateModulated.py:105]: we have explicitly simulating -> we explicitly simulated

File: tutorials/sim/plot_SimulatePeriodic.py [hunk 5]
```diff
@@ -126,65 +136,169 @@
...
+
+# Simulate a bursty oscillation, with a higher probability of entering bursts
+more_bursts = sim_bursty_oscillation(n_seconds, fs, osc_freq,
+                                     enter_burst=0.4, leave_burst=0.1)
 
 ###################################################################################################
+
+# Plot the simulated burst signals
+plot_time_series(times, short_burst, xlim=[0, n_seconds], title='Shorter Burst')
+plot_time_series(times, more_bursts, xlim=[0, n_seconds], title='More Bursts')
```
Comment (ryanhammonds) [inline tutorials/sim/plot_SimulatePeriodic.py:197]: These double plots render a bit weird in the same cell. Maybe separate into different cells or use custom axes.

File: tutorials/sim/plot_SimulateTransients.py [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+###################################################################################################
+
+# Plot the simulated synaptic kernel
+plot_time_series(None, kernel, figsize=(4, 3))
+
+###################################################################################################
+# Simulating Action Potentials
+# ----------------------------
+#
+# There is also the # :func:`~.sim_action_potential` function for simulating action potentials.
```
Comment (ryanhammonds) [inline tutorials/sim/plot_SimulateTransients.py:155]: \# :func: -> :func:

**Review by ryanhammonds — COMMENTED** [review]
This is great! Thanks for updating these. I pushed an update that was required so I could build the site locally. I left minor comments - I'll push some updates fixing typos / etc.

**Comment by TomDonoghue** [comment]
Thanks for checking through @ryanhammonds! Sorry I was a bit sloppy on the last checks for typos, etc - I should have done another sweep here. 

If you don't mind doing some direct edits for the typos, etc, then please go for it! Otherwise, if you prefer I fix things, and/or if you need anything from me, just let me know!

**Comment by ryanhammonds** [comment]
No problem! I went ahead and pushed edits for typos/etc. This should be ready to merge if you're happy with it.

**Comment by TomDonoghue** [comment]
@ryanhammonds - awesome, thank you! Fixes look good - merging this in now!

**Pull request merged**